### ISSUE TITLE
Datenstruktur bei Eisdeale

### ISSUE THREAD

**Issue by Elliiiii123** [JirkaDellOro/EIA2-Inverted#449]
Ich habe eine frage zu der EIS Deale. Wir sollen ja wie bei der Cocktailbar mit einer "dynamischen" Datenstruktur arbeiten. Mir ist allerdings nicht ganz klar, wie das mit einem Canvas, anstatt der in der Cocktailbar verwendeten HTML Elementre Funktionieren soll. Ich kann mir vorstellen, dass man durch die Datenstruktur die Preise und Beschriftungen der Eissorten ändern kann aber das auf dem Canvas gemalte muss ja mehr oder weniger fest sein. Also sollen wir jetzt wie bei der Cocktail bar nur die Namen und Preise für die Eissorten in diese Datenstruktur machen? und sollen wir auch diese create multiple Funktion nutzen um HTML Elemente zu erzeugen? Weil wir brauchen ja garnicht umbedingt HTML Elemente. Finde die Aufgabenstellung in der Hinsicht noch ein bisschen verwirrend.

**Comment by KohlerAl** [comment]
Genau, du bist auf jeden Fall in eine gute Richtung unterwegs. Anstatt den HTML-Elementen werden dann aber zum Beispiel Instanzen von einer Klasse erstellt. Im Array kannst du den Namen, die Farbe, den Preis etc. für alle Eissorten, die du möchtest speichern. Beim Auslesen kannst du die Daten dann verarbeiten, zum Beispiel in einer Klasse, die dann als Eiscreme-Behälter in der Eisdiele oder als Eiskugel dargestellt werden :)
